src/python/pants/option/global_options.py

```diff
@@ -273,6 +273,16 @@ def register_bootstrap_options(cls, register):
       help='Create a new pantsd server, and use it, and shut it down immediately after. '
            'If pantsd is already running, it will shut it down and spawn a new instance (Beta)')
 
+    # NB: We really don't want this option to invalidate the daemon, because different clients might have
+    # different needs. For instance, an IDE might have a very long timeout because it only wants to refresh
+    # a project in the background, while a user might want a shorter timeout for interactivity.
+    register('--pantsd-start-invocation-timeout', advanced=True, type=float, default=60.0, daemon=False,
+             help='The maximum amount of time to wait for the invocation to start until '
+                  'raising a timeout exception. '
+                  'Because pantsd currently does not support parallel runs, '
```

Comment:
Note that this will not actually give you a new line. You here are doing implicit string concatenation, which is the implicit version of this same thing:

```python
"this str " + "is all " + "combined " + "into one thing!"
```

The wrapping is solely useful for source code to be more readable. For the actual `str`, it's all one line, and then we rely on the terminal to do soft wrapping for us, which allows people to resize their terminal how they want.

Reply:
Yeah, implicit string concatenation is intentional, it's all over the options :)